strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/security/authorization/AuthorizationConfigControllerTestIT.java

```diff
@@ -170,7 +174,8 @@ void roleShouldNotBeDeleted(String acceptHeader)
 
     @ParameterizedTest
     @MethodSource("privilegesProvider")
-    void privilegesToAnonymousShouldBeAdded(String privilegeName, String acceptHeader) {
+    void privilegesToAnonymousShouldBeAdded(String privilegeName, String acceptHeader)
```

Comment:
Cosmetic: Could you please, follow the project's coding convention for the method parameters?